qa-mickey.planx-pla.net/manifest.json

```diff
@@ -7,32 +7,32 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "ambassador": "quay.io/datawire/ambassador:1.4.2",
-    "arborist": "quay.io/cdis/arborist:2021.10",
-    "aws-es-proxy": "quay.io/cdis/aws-es-proxy:v1.3.1",
-    "cohort-middleware": "quay.io/cdis/cohort-middleware:feat_init",
-    "dashboard": "quay.io/cdis/gen3-statics:2021.10",
-    "fence": "quay.io/cdis/fence:2021.10",
+    "ambassador": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ambassador:1.4.2",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.10",
+    "aws-es-proxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/aws-es-proxy:v1.3.1",
+    "cohort-middleware": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/cdis/cohort-middleware:feat_init",
```

Comment:
the ECR path is this `707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/<repo-name>` not this `707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/cdis/<repo-name>`

Also can you check if the image for `cohort-middleware` is in ECR?

Reply:
ah i see, will make the change. Also how do i where ecr images are held for cohort-middleware?

Reply:
I have run the command to copy cohort-middleware to ECR.